bodo/tests/test_df_lib/test_end_to_end.py

```diff
@@ -753,43 +759,52 @@ def test_set_df_column(datapath, index_val):
     df.index = index_val[: len(df)]
     bdf = bd.from_pandas(df)
```

Comment:
I'd say the context manager should wrap all the bodo.pandas operation so including this line.  In the other tests the pandas stuff at the beginning was also included and that is okay to keep everything consistent.